migrations/sql/V2022.02.22.13.20__update_preambule_text_column_to_existing_draft_permit_amendment_permit.sql

```diff
@@ -5,7 +5,7 @@ from (select distinct pa.permit_amendment_id, case when atc.application_type_cod
 	  where pa.permit_amendment_status_code = 'DFT' and
       		pa.is_generated_in_core = true and
             pa.preamble_text is NULL and
-            pa.permit_id = nai.permit_id and
+            pa.now_application_guid = nai.now_application_guid and
```

Comment:
migration files cannot be edited, you must create a new one with a new script.

Reply:
Wondering about that. Creating a new one.